scrapy/http/response/text.py

```diff
@@ -18,6 +18,7 @@
 from scrapy.http.response import Response
 from scrapy.utils.python import memoizemethod_noargs, to_unicode
 from scrapy.utils.response import get_base_url
+from scrapy.utils.decorators import deprecated
```

Comment:
We aim for alphabetical order:

```suggestion
from scrapy.utils.decorators import deprecated
from scrapy.utils.python import memoizemethod_noargs, to_unicode
from scrapy.utils.response import get_base_url
```